spyder/plugins/switcher/widgets/switcher.py

```diff
@@ -0,0 +1,472 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright © Spyder Project Contributors
+# Licensed under the terms of the MIT License
+# (see spyder/__init__.py for details)
+
+"""Switcher Main Widget."""
+
+# Standard library imports
+import os
+import sys
+
+# Third party imports
+from qtpy.QtCore import (QEvent, QObject, QSize, QSortFilterProxyModel, Qt,
+                         Signal, Slot, QModelIndex)
+from qtpy.QtGui import QStandardItem, QStandardItemModel, QTextDocument
+from qtpy.QtWidgets import (QAbstractItemView, QApplication, QDialog,
+                            QLineEdit, QListView, QListWidgetItem, QStyle,
+                            QVBoxLayout)
+
+# Local imports
+from spyder.config.base import _
+from spyder.config.manager import CONF
+from spyder.config.utils import is_ubuntu
+from spyder.plugins.switcher.widgets.proxymodel import SwitcherProxyModel
+from spyder.plugins.switcher.widgets.switcheritem import SwitcherItem, SwitcherBaseItem, SwitcherSeparatorItem
```

Comment:
```suggestion
from spyder.plugins.switcher.widgets.switcheritem import (
    SwitcherItem, SwitcherBaseItem, SwitcherSeparatorItem)
```

This line is too long.